data/modelCuration/transRxnNewGPR/TransRxnNewGPR.tsv

```diff
@@ -0,0 +1,29 @@
+r_1183	(YCL025C or YKR039W or YOR348C or YPL265W)	https://www.yeastgenome.org/locus/S000000272; yeastgenome.org/locus/S000005380 https://www.yeastgenome.org/locus/S000002453	https://www.uniprot.org/uniprot/P38084; https://www.uniprot.org/uniprot/P41815; https://www.uniprot.org/uniprot/P38967	https://biocyc.org/gene?orgid=YEAST&id=G3O-29037	https://www.genome.jp/entry/sce:YDR046C		YBR068C,  YDR046C and YOL020W could not transport alanine
+r_1218	(YCL025C or YDR508C or YKR039W)	https://www.yeastgenome.org/locus/S000000273; https://www.yeastgenome.org/locus/S000002453	https://www.uniprot.org/uniprot/P38085; https://www.uniprot.org/uniprot/P41815		https://www.genome.jp/entry/sce:YBR069C		YBR069C or YDR046C could not transport threonine
+r_1238	YKR039W	https://www.yeastgenome.org/locus/S000002453	https://www.uniprot.org/uniprot/P04817		https://www.genome.jp/entry/sce:YEL063C		YEL063C could not transport orntithine
+r_1805	YBR241C	https://www.yeastgenome.org/locus/S000003072	https://www.uniprot.org/uniprot/P53142		https://www.genome.jp/entry/sce:YEL063C		YGL104C involved in protein sorting, not glucose transport
+r_1910	(YDL245C or YDR342C or YDR343C or YDR345C or YEL069C or YFL011W or YHR092C or YHR096C or YJL214W or YJL219W or YJR158W or YNR072W or YOL156W)	https://www.yeastgenome.org/locus/S000001136	https://www.uniprot.org/uniprot/P32465; https://www.uniprot.org/uniprot/P23585		https://www.genome.jp/entry/sce:YHR094C		YHR094C and  YMR011W could not transport sorbose
+r_2105	YDL245C	https://www.yeastgenome.org/locus/S000002404; https://www.yeastgenome.org/locus/S000003966	https://www.uniprot.org/uniprot/P23900		https://www.genome.jp/entry/sce:YLL043W		YLL043W could not transport xylitol
+r_4168	YBR241C	https://www.yeastgenome.org/locus/S000003072	https://www.uniprot.org/uniprot/P53142		https://www.genome.jp/entry/sce:YGL104C		YGL104C involved in protein sorting, not sugar transport
+r_4169	YBR241C	https://www.yeastgenome.org/locus/S000003072	https://www.uniprot.org/uniprot/P53142		https://www.genome.jp/entry/sce:YGL104C		YGL104C involved in protein sorting, not sugar transport
+r_4592	YOL130W	https://www.yeastgenome.org/locus/S000003423	https://www.uniprot.org/uniprot/P06775		https://www.genome.jp/entry/sce:YGR191W		YGR191W could not transport Mg
+r_3961	(YER053C or YJR077C)	https://www.yeastgenome.org/locus/S000004340	https://www.uniprot.org/uniprot/Q06143		https://www.genome.jp/entry/sce:YLR348C		YLR348C transport dicarboxylic acids not phosphate
+r_1684	YGL077C	https://www.yeastgenome.org/locus/S000005687	https://www.uniprot.org/uniprot/Q12412		https://www.genome.jp/entry/sce:YOR161C		No evidence shows YOR161C could transport choline
+r_1101	YDL210W	https://www.yeastgenome.org/locus/S000004340	https://www.uniprot.org/uniprot/Q06143		https://www.genome.jp/entry/sce:YLR348C		YLR348C is located in mitochondrion
```

Comment:
Please check this line. You are removing the `YOR348C` rather than the  `YLR348C`. `YOR348C` is not a mitochondria protein